bigtable/google/cloud/bigtable/client.py

```diff
@@ -140,8 +140,7 @@ def _table_data_client(self):
         :rtype: :class:`.bigtable_v2.BigtableClient`
         :returns: A BigtableClient object.
         """
-        return bigtable_v2.BigtableClient(channel=self._channel,
-                                          credentials=self._credentials)
+        return bigtable_v2.BigtableClient(channel=self._channel)
```

Comment:
Why are we not forwarding through the credentials here?  See for instance the [Spanner client](https://github.com/zakons/google-cloud-python/blob/35d3199f654da56d0a3d7de1196b7f70eed50460/spanner/google/cloud/spanner_v1/client.py#L148-L155)